core/blockchain.go

```diff
@@ -2105,10 +2105,19 @@ func (bc *BlockChain) insertChain(chain types.Blocks, verifySeals bool) (int, er
 		if parent == nil {
 			parent = bc.GetHeader(block.ParentHash(), block.NumberU64()-1)
 		}
-		statedb, err := state.New(parent.Root, bc.stateCache, bc.snaps)
-		if err != nil {
-			return it.index, err
+		var statedb *state.StateDB
+		if ParallelTxMode {
```

Comment:
It is not necessary to add a new concept: BaseSlotDB.
We can mark the StateDB.paralleMode = true in StateProcessor.resetParallelState()